src/conn.c

```diff
@@ -525,6 +526,8 @@ _processInfo(natsConnection *nc, char *info, int len)
         s = nats_JSONGetULong(json, "client_id", &(nc->info.CID));
     if (s == NATS_OK)
         s = nats_JSONGetStr(json, "nonce", &(nc->info.nonce));
+    if (s == NATS_OK)
+        s = nats_JSONGetStr(json, "client_ip", &(nc->info.clientIP));
 
 #if 0
```

Comment:
You leave these in?

Reply:
Sorry, not sure I understand the question

Reply:
The #if 0 you mean?

Reply:
Yes, common to leave those in for folks to easily debug stuff?

Reply:
Been so long since I have done C ;)

Reply:
That was on old vestige of debugging the parsing of the INFO. I will remove.